polytope/polytope.py

```diff
@@ -772,21 +775,22 @@ def is_convex(reg, abs_tol=ABS_TOL):
     outer = envelope(reg)
     if is_empty(outer):
         # Probably because input polytopes were so small and ugly..
-        return False,None
+        return False, None
 
-    Pl,Pu = reg.bounding_box
-    Ol,Ou = outer.bounding_box
+    Pl, Pu = reg.bounding_box
+    Ol, Ou = outer.bounding_box
 
-    bboxP = np.hstack([Pl,Pu])
-    bboxO = np.hstack([Ol,Ou])
+    bboxP = np.hstack([Pl, Pu])
+    bboxO = np.hstack([Ol, Ou])
 
-    if sum(abs(bboxP[:,0] - bboxO[:,0]) > abs_tol) > 0 or \
-    sum(abs(bboxP[:,1] - bboxO[:,1]) > abs_tol) > 0:
-        return False,None
+    if sum(abs(bboxP[:, 0] - bboxO[:, 0]) > abs_tol) > 0 or \
```

Comment:
I do not support adding any indentation to the body of the selection construct. Main reasons:

1. avoid unnecessary indentation: in a more typical case, the body will have more lines than the conditional expression. Extra indentation will then result in all those lines being indented.
2. inconsistent with the pattern of 4 spaces indent per level. The eye is trained to follow this indentation. Variations can confuse the eye and slow down reading / skimming of code.

The reason for the extra indentation within the multi-line expression is treating conceptually as a lower-level construct, the same way that we indent within a function's body. I had not thought of the interpretation of `or` as being part of the `sum` (another reason to place the `or` at the end of the preceding line). In any case, this can be addressed as follows:

```python
if (
        sum(abs(bboxP[:, 0] - bboxO[:, 0]) > abs_tol) > 0
        or sum(abs(bboxP[:, 1] - bboxO[:, 1]) > abs_tol) > 0):
    return False, None
```

or

```python
if (
        sum(abs(bboxP[:, 0] - bboxO[:, 0]) > abs_tol) > 0 or
        sum(abs(bboxP[:, 1] - bboxO[:, 1]) > abs_tol) > 0):
    return False, None
```

(Had vertical disjunctions in the style of TLA been available in Python, I would have preferred the first version.)